sdk/python/kfp/v2/google/client/schedule.py

```diff
@@ -145,9 +152,9 @@ def _create_from_pipeline_dict(
       display_name=pipeline_display_name,
   )
 
-  project_location_path = 'projects/{}/locations/{}'.format(project_id, region)
-  scheduled_job_full_name = project_location_path + '/jobs/' + schedule_name
-  service_account_email = project_id + '@appspot.gserviceaccount.com'
+  project_location_path = 'projects/{}/locations/{}'.format(project_id, app_engine_region)
```

Comment:
`project_location_path` of cloud scheduler is built from app_engine_region. This `project_location_path` variable is used just for creating scheduler job. So it does not make side-effect for creating cloud functions and vertex pipeline.